code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -105,6 +105,14 @@
 /obj/machinery/atmospherics/proc/destroy_network()
 	return
 
+/// This should only be called by SSair as part of the rebuild queue.
+/// Handles rebuilding pipelines after init or they've been changed.
+/obj/machinery/atmospherics/proc/rebuild_pipes()
+	var/list/targets = get_rebuild_targets()
+	rebuilding = FALSE
+	for(var/datum/pipeline/build_off as anything in targets)
+		build_off.build_pipeline(src)
+
```

Comment:
I think the comment about what build pipeline does is valuable, but then again I'm the one who wrote it so